src/connection/mod.rs

```diff
@@ -45,10 +45,12 @@ pub fn create(s: &str, i: Interface, sc: Security, p: &str) -> Result<Connection
         path: "/org/freedesktop/NetworkManager/ActiveConnection/187".to_string(),
         id: "resin_io".to_string(),
         uuid: "3c8e6e8b-b895-4b07-97a5-bbc192c3b436".to_string(),
-        ssid: "resin_io".to_string(), /* device: "wlp4s0".to_string(),
-                                       * interface: Interface::WiFi,
-                                       * security: Security::WPA2,
-                                       * state: ConnectionState::Activated, */
+        ssid: "resin_io".to_string(),
```

Comment:
@lurch the functions are populated with dummy data to ensure the doc tests pass whilst we are implementing all the functions.